File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;

    do
    {
        // Get height
        n = get_int("How tall should the pyramids be? (between 1 and 8): ");
    }
    // Keep asking for height until an input between 1-8 is given
    while (n > 8 || n < 1);

    // Pyramid ow counter
    for (int i = 0; i < n; i++)
    {
        for (int j = n - 1; j > i; j--)
        {
            printf(" ");
        }
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-less/mario.py
from cs50 import get_int


while True:
    try:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            break
    except ValueError:
            print("That's not an integer!")


for i in range(1, height + 1):
    print("#" * i)